src/commands/sites/create.js

```diff
@@ -152,6 +153,10 @@ class SitesCreateCommand extends Command {
       }
     }
 
+    if (flags.json) {
+      this.logJson(pick(site, ["id","state","plan","name","custom_domain","domain_aliases","url","ssl_url","admin_url","screenshot_url","created_at","updated_at","user_id","ssl","force_ssl","managed_dns","deploy_url","account_name","account_slug","git_provider","deploy_hook","capabilities","id_domain"]))
```

Comment:
Does this log out env vars of the site?

Can't remember where that was in the object but we want to redact those

Reply:
@DavidWells ENV vars are part of `build_settings` which is excluded from the output.